ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_splitting_arrays.py

```diff
@@ -0,0 +1,68 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+from hypothesis import strategies as st
+
+
+@st.composite
+def _arrays_idx_n_dtypes(draw):
+    num_dims = draw(st.shared(helpers.ints(min_value=1, max_value=4), key="num_dims"))
+    num_arrays = draw(
+        st.shared(helpers.ints(min_value=2, max_value=4), key="num_arrays")
+    )
+    common_shape = draw(
+        helpers.lists(
+            arg=helpers.ints(min_value=2, max_value=3),
+            min_size=num_dims - 1,
+            max_size=num_dims - 1,
+        )
+    )
+    unique_idx = draw(helpers.ints(min_value=0, max_value=num_dims - 1))
+    unique_dims = draw(
+        helpers.lists(
+            arg=helpers.ints(min_value=2, max_value=3),
+            min_size=num_arrays,
+            max_size=num_arrays,
+        )
+    )
+    xs = list()
+    xa = list()
+    input_dtypes = draw(
+        helpers.array_dtypes(available_dtypes=draw(helpers.get_dtypes("valid")))
+    )
+    for ud, dt in zip(unique_dims, input_dtypes):
+        x = draw(
+            helpers.array_values(
+                shape=common_shape[:unique_idx] + [ud] + common_shape[unique_idx:],
+                dtype=dt,
+            )
+        )
+        xs.append(x)
+        xa.append(x)
+    return xs, xa, unique_idx
+
+
+# split
+@handle_frontend_test(
+    fn_tree="numpy.split",
+    dtype_and_x=_arrays_idx_n_dtypes(),
+)
+def test_numpy_split(
+    as_variable,
+    dtype_and_x,
+    native_array,
+    num_positional_args,
+):
+    indices_or_sections, ary, axis = dtype_and_x

```

Comment:
There already exist helpers for generating arrays and indices; I suggest you check out the `hypothesis_helpers` folder inside the `ivy_tests` folder. If I'm not wrong, the function is called `array_indices_axis.`